pkg/scheduler/framework/interface.go

```diff
@@ -105,6 +106,27 @@ const (
 	MaxTotalScore int64 = math.MaxInt64
 )
 
+// PodsToActivateKey is a reserved state key for stashing pods that will be
+// moved to activeQ if they're present in unschedulableQ or backoffQ.
+var PodsToActivateKey StateKey = "kubernetes.io/pods-to-activate"
+
+// PodsToActivate stores pods to be activated.
+type PodsToActivate struct {
+	sync.Mutex
+	// Map is keyed with namespaced pod name, and valued with the pod.
```

Comment:
why do you need the pod, if we already have it in the unschedulable/backoff queues?

Reply:
well, we only need a namespaced name here, but the current interface in unschedulable/backoff queues only accepts `pod`, so have to adapt to it; otherwise adjusting the interface isn't trivial.